nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-client-service/pom.xml

```diff
@@ -127,8 +133,113 @@
             <version>5.6.8</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-avro-record-utils</artifactId>
+            <version>1.7.0-SNAPSHOT</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-schema-registry-service-api</artifactId>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-all</artifactId>
+            <scope>test</scope>
+        </dependency>
     </dependencies>
 
+    <build>
```

Comment:
Probably best to remove the groovy and Jacoco stuff, let's get a discussion going on the dev mailing list about code coverage?

Reply:
I'm leaving in the helper plugin for now because for some reason, it won't even detect the groovy test source without it. I'll remove it if you have any suggestions on how to fix that.

Reply:
If there is nothing in `src/test/java`, the Groovy tests won't be detected unless a plugin references them directly. In this case, the `build-helper-maven-plugin` is accomplishing that. In other locations, the `maven-compiler-plugin` is set to use `groovy-eclipse-compiler` to achieve the same result.

Reply:
@alopresto thanks for explaining that. I just added a .gitignore file into src/test/java and that did the trick.